aws/addAppSettings.sh

```diff
@@ -33,11 +34,11 @@ while getopts ":hs:u" opt; do
             UPDATE_APPSETTINGS="true"
             ;;
         \?)
-            echo -e "\nInvalid option: -${OPTARG}"
+            cat <<< "Invalid option: -${OPTARG}" >&2
             usage
             ;;
         :)
-            echo -e "\nOption -${OPTARG} requires an argument"
+            cat <<< "Option -${OPTARG} requires an argument" >&2
```

Comment:
Use of stderr or error yes. Use of herestring (<<<) is only for case where text is then piped to another command. So in this case you would leave the echo but redirect to stderr.